_posts/2020-05-18-mirror-kafka-eventhub.md

```diff
@@ -93,6 +93,11 @@ Where `<eventhubs-namespace>` has to be replaced with the actual Event Hub names
 
 ### Configure Kafka Mirror Maker
 
+When mirroring messages from a source Apache Kafka cluster to a target one, the usual paradigm is deploying in the Kafka Mirror Maker instance alongside the target cluster.
+It means consuming remotely and producing locally.
+In this integration scenario, it is not possible because there is no way to deploy Kafka Mirror Maker alongside the Event Hub namespace that is the target cluster.
```

Comment:
Would it be worth mentioning that if the reader really wants she/he can run Mirror Maker with Strimzi on AKS to run it in the same cloud? I wouldn't change the blog post because of it, but maybe worth mentioning as alternative? Just thinking ...

Reply:
I added a sentence about it.